mix.exs

```diff
@@ -12,7 +12,7 @@ defmodule VegaLite.MixProject do
   end
 
   def application do
-    []
+    [extra_applications: [:wx]]
```

Comment:
Is this alright? Doesn't seem to cause problems if `:wx` is not available.

Reply:
We don't want to list it here because it will load :wx in prod even if it is not available.

Reply:
```
warning: :wx.new/0 defined in application :wx is used by the current application but the current application does not depend on :wx. To fix this, you must do one of:

  1. If :wx is part of Erlang/Elixir, you must include it under :extra_applications inside "def application" in your mix.exs

  2. If :wx is a dependency, make sure it is listed under "def deps" in your mix.exs

  3. In case you don't want to add a requirement to :wx, you may optionally skip this warning by adding [xref: [exclude: [:wx]]] to your "def project" in mix.exs

  lib/vega_lite/wx_viewer.ex:28: VegaLite.WxViewer.init/1
```

Is the 3rd option ok then?

Reply:
Yes!